06-gitops.md

```diff
@@ -67,8 +67,8 @@ GitOps allows a team to author Kubernetes manifest files, persist them in their
    az acr import --source docker.io/fluxcd/flux:1.19.0 -n $ACR_NAME
    az acr import --source docker.io/weaveworks/kured:1.4.0 -n $ACR_NAME
    az acr import --source quay.io/k8scsi/csi-node-driver-registrar:v1.2.0 -n $ACR_NAME
-   az acr import --source docker.io/deislabs/secrets-store-csi:v0.0.11 -n $ACR_NAME
-   az acr import --source quay.io/k8scsi/livenessprobe:v1.1.0 -n $ACR_NAME
+   az acr import --source us.gcr.io/k8s-artifacts-prod/csi-secrets-store/driver:v0.0.16 -n $ACR_NAME
```

Comment:
Should these changes should also be made in [inner-loop-scripts/azcli/cluster-deploy.azcli](inner-loop-scripts/azcli/cluster-deploy.azcli)

Reply:
I usually don't maintain those sh files. I've never ran them, I think @v-fearam maintains them. No idea if they are still in sync with the bigger work here.

Reply:
I didn't spend a second since I started the new project. If some change was done, for sure is out of sync.

Reply:
I'll update the azcli file, they should indeed be made there.  Good catch

Reply:
Updated the `.azcli` file.  Going to leave the `.sh` one behind.  If that is something we want to update, I'd suggest a story to bring them up to date.  Right now they are labeled as "something we used when we built it" not as part of any specific instructions to deploy, IIRC.